src/views/include/list/added.html

```diff
@@ -0,0 +1,44 @@
+{% if (add_team and add_team.length > 0)
+    or (add_repo and add_repo.length > 0)
+    or (add_member and add_member.length > 0) %}
+
+    <table class="govuk-table">
```

Comment:
Could you use [Summary List nunjucks macro](https://design-system.service.gov.uk/components/summary-list/) instead of manually writing `html`? 

Because we have logic to render multiple objects I'm not sure if using the macro is possible though...

Reply:
We have used summary in the check your answers page, we could reuse it but lets have another UX day to review it